src/css/skills.scss

```diff
@@ -0,0 +1,130 @@
+@import './core/constants';
+@import './core/mixins';
+
+#skills {
+  padding: 35px 70px;
+  min-height: 600px;
+  background-image: url('https://static.wixstatic.com/media/05e3dc_cc646f530d6846828d53dd3216ec8023~mv2.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
```

Comment:
Very smart I like it, but would be better to use short hand so you don't have to write so many lines:
https://developer.mozilla.org/en-US/docs/Web/CSS/background